pkg/pool-stable/test/ComposableStablePoolProtocolFees.test.ts

```diff
@@ -281,7 +281,7 @@ describe('ComposableStablePoolProtocolFees', () => {
 
       const PREMINTED_BPT = fp(9e9); // This is the BPT normally stored in the Pool's accounting at the Vault
 
-      // We want a Pool that is relatively balaced so that, with a reasonably high amplification factor, each token has
+      // We want a Pool that is relatively balanced so that, with a reasonably high amplification factor, each token has
       // similar prices
```

Comment:
```suggestion
      // We want a Pool that is relatively balanced so that, with a reasonably high amplification factor, each token
      // has similar prices
```
Change will go over the line length limit.